tools/build-templated-pages/src/features.rs

```diff
@@ -58,53 +38,88 @@ fn parse_features(panic_on_missing: bool) -> Vec<Feature> {
             if key == "default" {
                 None
             } else {
-                let name = key
-                    .as_repr()
-                    .unwrap()
-                    .as_raw()
-                    .as_str()
-                    .unwrap()
-                    .to_string();
-                if let Some(description) = key.leaf_decor().prefix() {
-                    let description = description.as_str().unwrap().to_string();
-                    if !description.starts_with("\n# ") || !description.ends_with('\n') {
-                        panic!("Missing description for feature {name}");
+                let key_name = get_key_name(key);
+                let key_prefix = key.leaf_decor().prefix();
+                if what_to_run.contains(Command::CHECK_MISSING) {
+                    if let Some(description) = key_prefix {
+                        create_feature(&default, key_name, description)
+                    } else {
+                        panic!("Missing description for feature {key_name}");
                     }
```

Comment:
```suggestion
                    match key_prefix {
                        Some(description) => create_feature(&default, key_name, description),
                        None => panic!("Missing description for feature {key_name}"),
                    }
```

I prefer a `match` in this scenario.